docs/contributing/create_issues.md

```diff
@@ -1,31 +1 @@
-
-The Islandora [issue queue](https://github.com/Islandora/documentation/issues) is maintained in the Islandora [Documentation](https://github.com/Islandora/documentation) repository on GitHub. Issues posted to the queue are reviewed weekly on the Islandora [Tech Call](https://github.com/Islandora/islandora-community/wiki/Tech-Call-Meetings-2022). Members of the Islandora community can then respond to posted issues by replying with comments, taking on the assignment to do the work described in the issue, or making pull requests relating to the issue.
-
-!!! Tip "Video version available"
-    The material in this tutorial is presented in our video, [Create a GitHub Issue](https://youtu.be/eHBIYlRxEtk).
-
-## Before you start
-
-- You will need to have an account on [GitHub](https://github.com/join). These are free to register and require only a name and email address.
-
-
-## How to create an issue in the Islandora issue queue
-
-1. Go to [https://github.com/Islandora/documentation/issues](https://github.com/Islandora/documentation/issues)
-2. Click on **New issue**:
-![New Issue button](../assets/create_issues_newissue.jpg)
-3. Select the type of issue you are creating, as this will provide you with a template to describe your issue. 
-![Bug report, Documentation, Feature Request, and Use case are presented as possible issue templates when opening a new issue.](../assets/create_issues_Issue_Templates.png)
-4. Fill in the information for your issue:
-    1. Give your issue a descriptive title following the text that is already provided in the template title, ex. _[BUG]._
-    2. Fill in the body of your issue under the **Write** tab. A template of questions will be provided based on the type of issue selected. Depending on your issue, you may need to: describe a bug you are seeing and how to reproduce it, describe how an existing feature could be improved, describe a new feature and how it should work, or describe documentation that needs to be written or expanded.
-    3. Use the built-in text editor to help format your issue in [Markdown](http://en.wikipedia.org/wiki/Markdown).
-    4. Use the **Preview** button to see how your issue will be published and ensure the formatting looks the way you want.
-    5. Click **Submit new issue** to add your issue to the queue.
-    6. Optionally, Add labels to your issue to assign it to available categories, such as "documentation" or "question." Click on as may labels as you like. When you click outside of the drop-down list, the selected labels will be applied to your issue.
-    
-    ![Gif walkthrough of templates](../assets/create_issues_Describing_Issue.gif) <img src="https://raw.githubusercontent.com/Islandora/documentation/3635b2d77681ce7da2fdbfd8f951921e97cc6acb/docs/assets/create_issues_Add_Labels.png" alt="Labels on the right sidebar of Github issue creation" width= 250 /></p>
+This content has been moved to [create-issues.md](create-issues.md).
```

Comment:
I realize this is a nit-pick,  but can you remove the `.md` from the link text? The extension doesn't appear in the URL once the site is built, making it appear inconsistent. See https://islandora.github.io/documentation/pr-preview/pr-2276/user-documentation/content_types/

Reply:
`This content has been moved to [create-issues](create-issues.md).`